README.md

```diff
@@ -22,16 +24,112 @@ The biometryService Interface 'IBiometryService' Implement the following method
 - Encryt
 - Decrypt
 
+As of now, this is the list of features available per platform.
+
+
+| Feature          | IOS | Android | UWP  |
+| ------------------ | ----- | --------- | ------ |
```

Comment:
you can probably align this

Reply:
Yeah tried, it's doing weird stuff when I do :(